packages/remediations/src/newSteps/selectPlaybook.scss

```diff
@@ -0,0 +1,13 @@
+.ins-c-remediations-can-be-remediated { 
+    color: var(--pf-global--success-color--100);
+    margin-bottom: 0px !important;
```

Comment:
Can you try using `pf-u-mb-0` and other spacing classes directly on the elements? It might work.

Reply:
Yep, it works. Thanks!